source/stylesheets/refills/_modal.scss

```diff
@@ -38,8 +38,8 @@
 
   .modal-fade-screen { // overlay
     @include transition(opacity 0.25s ease);
-    @include position(fixed, 0px 0px 0px 0px);
-    background: rgba(0,0,0, 0.85);
+    @include position(fixed, 0 0 0 0);
+    background: rgba(0, 0, 0, 0.85);
```

Comment:
- Per our style guide, we should use `background-color` here, rather then just `background`
- This can also be simplified to `rgba(#000, 0.85);`